Library/Homebrew/os/mac.rb

```diff
@@ -9,9 +9,9 @@
 
 module OS
   module Mac
-    extend self
+    module_function
 
-    ::MacOS = self # compatibility
+    ::Object.const_set(:MacOS, self) # compatibility
```

Comment:
(just so we can start killing the `Exclude` lines in the Rubocop files in favour of inline comments and avoid a similar problem in the same file)